README.md

```diff
@@ -359,10 +358,30 @@ To release new version run
 grunt uglify bump
 
 ```
-
 ### Testing
 
-Open `test/SpecRunner.html` in your web browser to run Jasmine test suite.
+
```

Comment:
Does it apply to PalindromDOM or Palindrom in general?

Above you still have links to `test/SpecRunner.html` which no longer exists